forJIS7/src/main/java/JavaHomework/Lectures_10/Collections/Shop/Service/ShopServiceImplementation.java

```diff
@@ -0,0 +1,38 @@
+package JavaHomework.Lectures_10.Collections.Shop.Service;
+
+import JavaHomework.Lectures_10.Collections.Shop.Model.Product;
+import static JavaHomework.Lectures_10.Collections.Shop.Repository.Shop.*;
+
+public class ShopServiceImplementation implements ShopService {
+
+    @Override
+    public void addProduct(Product product) {
+        getProductList().add(product);
+    }
+
+    @Override
+    public void deleteProduct(int productForDelete) {
+        getProductList().remove(productForDelete);
+    }
+
+    @Override
+    public void showAllProduct() {
+        for (Product productEach:getProductList()) {
+            System.out.println(productEach);
+        }
+    }
+
+    @Override
+    public void fiendProductByName(String productName) {
```

Comment:
`findProductByName` you have syntax mistake.
And if method start with `find` I expected that this method get me some objects, in your cases I want get Product. 
But you return void

Reply:
fixed